### PR TITLE
Add 'memory' page to 'system' menu

### PR DESCRIPTION
**Pull request by jcapona**
| Status  | Ticket/Issue |
| :---: | :--: |
| Ready | [Ticket](https://pi-top.atlassian.net/browse/OS-1215) |

#### Main changes
- Adds 'memory' page in 'system' menu, with information about RAM & swap usage.

#### Screenshots (feature, test output, profiling, dev tools etc)

image

#### Other notes (e.g. implementation quirks, edge cases, questions / issues)
-

#### Manual testing tips
-

#### Tag anyone who definitely needs to review or help
-

### CODE REVIEW

File: pt_miniscreen/pages/system/memory.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+
+class Page(PageBase):
+    def __init__(self, size):
+        super().__init__(size=size)
+        self.reset()
+
+    def reset(self):
+        try:
+            memory = psutil.virtual_memory()
```
Comment (olivierwilkinson) [inline pt_miniscreen/pages/system/memory.py:28]: Since this is calculated once and not updated the memory doesn't change until reset is called again. I think `psutil.virtual_memory` and `psutil.swap_memory` need to be moved into the functions that are passed into the marquee text and progress bar hotspots.

**Review by olivierwilkinson — CHANGES_REQUESTED** [review]
Looks great! There is one issue about the memory and swap not being updated but it looks like a quick fix. 👍

**Review by olivierwilkinson — APPROVED** [review]
LGTM!
